server/models/Wallet.spec.js

```diff
@@ -274,9 +274,9 @@ describe("Wallet", () => {
         target_wallet_id: wallet.getId(),
       }]);
       const result = await wallet.hasTrust(
-        TrustRelationship.ENTITY_TRUST_TYPE.send,
-        wallet,
+        TrustRelationship.ENTITY_TRUST_REQUEST_TYPE.receive,
```

Comment:
As a result of my change on hasTrust method referencing the trust type parameter now, I also think this test should expect a "receive" request type (should not be sending an entity trust type). Since wallet2 (actor, receiver) is sending a receive request to wallet (target, sender).